src/stories/Blocks/material-manifestation-item/material-manifestation-item.scss

```diff
@@ -30,6 +32,7 @@
   display: flex;
   flex-wrap: wrap;
   gap: 6px;
+  height: fit-content;
```

Comment:
I don't this is relevant since you use the grid-template-rows to decide the height

Reply:
You're right! It's not necessary :) I removed it now.